languages/fr.json

```
@@ -14,11 +14,55 @@
         "rank-upgrade": {
           "name": "[Règle maison] Les sorts de convocation sont de Rang +1",
           "hint": "Avertissement : Je ne prends pas la responsabilité sur la manière dont cela peut affecter l'équilibre de votre partie. Cette règle considère que les sorts de convocation sont d'un rang de plus pour déterminer quelle créature est conjurée."
-      },
+        },
         "scale-to-max-summon-level-for-rank": {
           "name": "[Règle maison] Les sorts de convocation sont élevés au niveau Max",
           "hint": "Avertissement : Je ne prends pas la responsabilité sur la manière dont cela peut affecter l'équilibre de votre partie. Ceci utilise le code Élever au niveau Max du module PF2e Workbench pour élever les créatures convoquées par des sorts de convocation traditionnels (ex Convocation d'animaux etc.) aux stats d'une créature du plus haut niveau possible que ce rang de sort peut convoquer."
         }
+      },
+      "necromancer": {
+        "handle-living-graveyard-movement": {
+          "name": "[Necromancer] Handle Living Graveyard Movement",
+          "hint": "En cas d'activation, lorsqu'un Cimetière vivant est déplacé , il enverra un message dans la conversation. En cliquant dessus, cela provoquera l'apparition d'un rejeton du cimetière (comme prévu par les règles)"
```

```suggestion
          "hint": "En cas d'activation, lorsqu'un Cimetière vivant est déplacé , il enverra un message dans la conversation. En cliquant dessus, cela provoquera l'apparition d'un thrall du cimetière (comme prévu par les règles)"
```
I don't think there is an "official" translation for "thrall", but "rejeton" feels strange in this case - it would fit the word "spawn" better. Better left untranslated for now I think^^